python/src/MethodOfMomentsFactory_doc.i.in

```diff
@@ -14,22 +14,49 @@ Notes
 -----
 This method fits a scalar distribution to data of dimension 1, using the method of moments.
 
-Let :math:`(\vect{x}_1, \dots, \vect{x}_n)` denote the sample, :math:`F_{\vect{\theta}}`
+Let :math:`(\vect{x}_k)_{1 \leq k \leq n}` be
+the sample, :math:`F_{\vect{\theta}}`
 the  cumulative distribution function we want to fit to the sample,
-and :math:`\vect{\theta} \in  \Theta \subset\Rset^p`  its parameter vector.
+and :math:`\vect{\theta} \in  \Theta \subset\Rset^K`  its parameter vector.
 
-Let :math:`K` denote the number of parameters of the distribution and we assume that 
+Then :math:`K` denotes the number of parameters of the distribution and we assume that
```

Comment:
This sentence is not in the right place: the number of parameters is already used in the previous sentence.

Reply:
ok